### PR TITLE
Improve build guide for android

### PR DESCRIPTION
**Pull request by bitfriend**
Changed troubleshooting.md.
But it may be better to move it to setup.md.

### CODE REVIEW

File: docs/content/docs/getting-started/troubleshooting.md [hunk 1]
```diff
@@ -15,9 +15,19 @@ top = false
 
 The current system has a problem with the latest android native development kit (NDK), please downgrade to version r22.* - then things should be fine.
 
-## Emulator crash 
-
-Please use the `Pixel 4 API 32 x86` emulator for development and testing. Other Emulators - in particular 32bit variants - have shown to be problematic.
```
Comment (gnunicorn) [inline docs/content/docs/getting-started/troubleshooting.md:20]: why did you remove this line?

File: docs/content/docs/getting-started/troubleshooting.md [hunk 2]
```diff
@@ -15,9 +15,19 @@ top = false
...
+## Android Build in Windows
```
Comment (gnunicorn) [inline docs/content/docs/getting-started/troubleshooting.md:18]: I agree, let's move this into setup.md under its own section.

File: docs/content/docs/getting-started/troubleshooting.md (hunk 1 above)
Comment (bitfriend) [reply docs/content/docs/getting-started/troubleshooting.md:20]: I confirmed this app crashes on API 30 x86 sys image.
This is log.

```
An Observatory debugger and profiler on sdk gphone x86 is available at: http://127.0.0.1:64330/gW6Sj_Djv_s=/
backtrace:
      #00 pc 00056337  /apex/com.android.runtime/lib/bionic/libc.so (memmove_generic+71) (BuildId: 6e3a0180fa6637b68c0d181c343e6806)
      #01 pc 03b2e0dd  /data/app/~~u78G5a78FkKTJbUeA98QVQ==/org.effektio.app-N-3wlo1L43l7N1nPzf2kIA==/lib/x86/libeffektio.so (_$LT$alloc..string..String$u20$as$u20$core..clone..Clone$GT$::clone::hc7061aaa0585626c+93)     
      #02 pc 017ab6f8  /data/app/~~u78G5a78FkKTJbUeA98QVQ==/org.effektio.app-N-3wlo1L43l7N1nPzf2kIA==/lib/x86/libeffektio.so (effektio::api::auth::guest_client::_$u7b$$u7b$closure$u7d$$u7d$::h398d46d7ae7bf09a+232)        
      #03 pc 0146809b  /data/app/~~u78G5a78FkKTJbUeA98QVQ==/org.effektio.app-N-3wlo1L43l7N1nPzf2kIA==/lib/x86/libeffektio.so (_$LT$core..future..from_generator..GenFuture$LT$T$GT$$u20$as$u20$core..future..future..Future$GT$::poll::h6f52c8d370a78b94+139)
      #04 pc 01335c7a  /data/app/~~u78G5a78FkKTJbUeA98QVQ==/org.effektio.app-N-3wlo1L43l7N1nPzf2kIA==/lib/x86/libeffektio.so (effektio::api::api::__guest_client::_$u7b$$u7b$closure$u7d$$u7d$::_$u7b$$u7b$closure$u7d$$u7d$::h5f5df3ac21f40009+554)
      #05 pc 01465dbb  /data/app/~~u78G5a78FkKTJbUeA98QVQ==/org.effektio.app-N-3wlo1L43l7N1nPzf2kIA==/lib/x86/libeffektio.so (_$LT$core..future..from_generator..GenFuture$LT$T$GT$$u20$as$u20$core..future..future..Future$GT$::poll::h5d26a16f33e1deb2+139)
      #06 pc 017c7a83  /data/app/~~u78G5a78FkKTJbUeA98QVQ==/org.effektio.app-N-3wlo1L43l7N1nPzf2kIA==/lib/x86/libeffektio.so (_$LT$core..pin..Pin$LT$P$GT$$u20$as$u20$core..future..future..Future$GT$::poll::h5413346511c8a480+131)
      #07 pc 013340c3  /data/app/~~u78G5a78FkKTJbUeA98QVQ==/org.effektio.app-N-3wlo1L43l7N1nPzf2kIA==/lib/x86/libeffektio.so (effektio::api::api::FfiFuture$LT$T$GT$::poll::he5ae23a1be505d41+211)
      #08 pc 0134615e  /data/app/~~u78G5a78FkKTJbUeA98QVQ==/org.effektio.app-N-3wlo1L43l7N1nPzf2kIA==/lib/x86/libeffektio.so (effektio::api::api::__guest_client_future_poll::_$u7b$$u7b$closure$u7d$$u7d$::h685bdf1cbee1f0f1+126)
      #09 pc 01602bd4  /data/app/~~u78G5a78FkKTJbUeA98QVQ==/org.effektio.app-N-3wlo1L43l7N1nPzf2kIA==/lib/x86/libeffektio.so (std::panicking::try::do_call::h641803c7f2a04d8e+116)
      #10 pc 0161820e  /data/app/~~u78G5a78FkKTJbUeA98QVQ==/org.effektio.app-N-3wlo1L43l7N1nPzf2kIA==/lib/x86/libeffektio.so (__rust_try+46)
      #11 pc 015f0d5a  /data/app/~~u78G5a78FkKTJbUeA98QVQ==/org.effektio.app-N-3wlo1L43l7N1nPzf2kIA==/lib/x86/libeffektio.so (std::panicking::try::h82039628f9844fd6+234)
      #12 pc 017a4538  /data/app/~~u78G5a78FkKTJbUeA98QVQ==/org.effektio.app-N-3wlo1L43l7N1nPzf2kIA==/lib/x86/libeffektio.so (std::panic::catch_unwind::h55f235af222ad3de+72)
      #13 pc 0132fb92  /data/app/~~u78G5a78FkKTJbUeA98QVQ==/org.effektio.app-N-3wlo1L43l7N1nPzf2kIA==/lib/x86/libeffektio.so (effektio::api::api::panic_abort::hfdbfad59d9623070+82)
      #14 pc 016371bb  /data/app/~~u78G5a78FkKTJbUeA98QVQ==/org.effektio.app-N-3wlo1L43l7N1nPzf2kIA==/lib/x86/libeffektio.so (__guest_client_future_poll+75)
      #15 pc 0000436f  [anon:dart-codespace]
Lost connection to device.
```
For now, it is clear x86 is not working.
When rust api for file transfer is done, I can review this problem.

Comment (bitfriend) [reply docs/content/docs/getting-started/troubleshooting.md:20]: Or I can split this commit into the following:
1. Add new x86_64 build guide.
2. Remove x86 build guide, because it is not working now.
You can ignore 2nd one, if you think it is necessary to leave x86 build.

I will create new pull request for these?

Comment (gnunicorn) [reply docs/content/docs/getting-started/troubleshooting.md:20]: Ah. tehre's a typo in this line... it's supposed to be `x64` not `x86`...
